src/file_readers.py

```diff
@@ -0,0 +1,123 @@
+"""
+Notes
+-----
+This module contains the functions for audiobook_gen that read in the
+file formats that require for parsing than plain text (pdf, html, epub),
+as well as the preprocessing function for all input files.
+"""
+
+__all__ = ['preprocess', 'read_pdf', 'read_epub']
+
+import re
+
+from bs4 import BeautifulSoup
+from nltk import tokenize, download
+from textwrap import TextWrapper
+from stqdm import stqdm
+
+import src.config as cf
+
+download('punkt', quiet=True)
+wrapper = TextWrapper(cf.MAX_CHAR_LEN, fix_sentence_endings=True)
+
+
+def preprocess(file):
```

Comment:
What exactly are you preprocessing? A better name would be <verb describing what you're doing>_<what you're applying the verb to>. In this case, `preprocess_data` or `preprocess_text` or something like that.

Reply:
Renamed `preprocess` to `preprocess_text`.